server/schema/questions/createQuestions.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  required: ['type', 'name', 'problem_statement'],
+  properties: {
+    type: {
+      type: 'string',
+      enum: ['mcq', 'subjective'],
+    },
+    name: {
+      type: 'string',
+    },
+    problem_statement: {
+      type: 'string',
+    },
+    input_format: {
+      type: 'string',
+    },
+    output_format: {
+      type: 'string',
+    },
+    constraints: {
+      type: 'string',
+    },
+    options: {
+      type: 'string',
+      pattern: '^\\w+(?:(?:,\\w+)+|(?:\\s\\w+)+)$',
```

Comment:
```suggestion
```
This pattern is not working properly. For eg. it is giving invalid for `"[1,2,3]"` or `"["abc","def"]"`
We can omit this pattern thing here, as it would be a JSON array. and if we need to parse it, the JSON.parse() itself will throw an error.

Reply:
Ultimately, we need to store the JSON array, as it is, in the DB, when a question is created.
When judging a MCQ submission, we can simply call JSON.parse() on it, if its a JSON array..

Reply:
No, it will work properly, I have escaped backslash in javascript but that would actually be equivalent to ^\w+(?:(?:,\w+)+|(?:\s\w+)+)$'

Reply:
Alright let me remove this

Reply:
@NBNARADHYA try running this in console :
```
a=JSON.stringify(["abc","def"])
reg = new RegExp('^\\w+(?:(?:,\\w+)+|(?:\\s\\w+)+)$')
console.log(reg.test(a))
```